<launch>
    <arg name="open_rviz" default="true" />
    <arg name="open_reconfigure" default="true" />
    <arg name="pattern_following" default="false" />
    <arg name="rosplane_tuner" default="false" />

    <node name="xplane_wrapper" pkg="xplane_ros" type="xplane_ros_wrapper.py" output="screen" />
    <node name="ned_to_viz" pkg="xplane_ros" type="ned_to_viz.py" />
    <node name="xp_rosplane_viz" pkg="xplane_ros" type="rosplane_path_viz.py" />

    <group if="$(arg open_rviz)">
        <node type="rviz" name="rviz" pkg="rviz" args="-d $(find xplane_ros)/rviz/default.rviz" />
    </group>
    <!-- <node name="ned_to_viz" pkg="tf" type="static_transform_publisher" args="0 0 0 3.14 0 0 world map 100" /> -->

    <group if="$(arg pattern_following)">
        <include file="$(find rosplane_sim)/launch/xplane_fw.launch" />
    </group>

    <group if="$(arg rosplane_tuner)">
        <include file="$(find xplane_ros)/launch/rosplane_tuner.launch" />
    </group>

</launch>